c10/util/numa.cpp

```diff
@@ -0,0 +1,132 @@
+#include "c10/util/numa.h"
+
+C10_DEFINE_bool(caffe2_cpu_numa_enabled, false, "Use NUMA whenever possible.");
```

Comment:
@dzhulgakov @ilia-cher Do we actually need this flag? I'd much rather if we just always have NUMA enabled if it's available, saves a fiddly configuration flag.

Reply:
without it we will treat legacy protobufs as having all CPU ops pinned to NUMA node 0.

Reply:
hm, so right now it's used only in some benchmarks from what I can tell. How do you plan to eventually roll out it widely?

Reply:
@ilia-cher so simply removing the flag in all places would work?

Reply:
yes, there're a few places (allocator) that will result in extra calls but these calls should be no-ops (since numa node id is -1 by default), make sure to rerun tests though

Reply:
what are the tests? @ilia-cher